misc/nanovgutils/src/processors/nanovgexampleprocessor.cpp

```diff
@@ -58,14 +59,28 @@ void NanoVGExampleProcessor::process() {
     // are in the range of [0 canvassize]
     nvg.activate(outport_.getDimensions());
 
-    nvg.beginPath();
-    // Draw a circle with of radius 50 and x-y coordinates 150,150
-    nvg.circle(vec2(150, 150), 50);
-    nvg.closePath();
+    std::array<std::pair<vec2, vec4>, 4> circles{
+        std::pair{vec2{150, 150}, vec4{1.0f, 0.0f, 0.0f, 0.6f}},
+        std::pair{vec2{200, 150}, vec4{0.0f, 1.0f, 0.0f, 0.6f}},
+        std::pair{vec2{175, 100}, vec4{0.0f, 0.0f, 1.0f, 0.6f}},
+        std::pair{vec2{175, 175}, vec4{0.5f, 0.5f, 0.5f, 0.2f}}};
+
+    for (int i{0}; i < 4; ++i) {
+        nvg.beginPath();
+        // Draw a circle
+        nvg.circle(circles[i].first, (5 - i) * 30);
+        nvg.closePath();
+
+        // Set fill color to red
+        nvg.fillColor(circles[i].second);
+        nvg.fill();  // tell NVG to fill current shape
+    }
 
-    // Set fill color to red
-    nvg.fillColor(vec4(1, 0, 0, 1));
-    nvg.fill();  // tell NVG to draw current shape
+    nvg.fillColor(fontProperty_.getFontColor());
+    nvg.fontSize(fontProperty_.getFontSize());
+    nvg.fontFace(fontProperty_.getFontFace());
+    nvg.textAlign(fontProperty_.getFontAligntment());
```

Comment:
I would put this in a until function: 
```
void setFont(NanoVGContext &nvg, const NanoVGFontProperty &property) {
    nvg.fillColor(property.getFontColor());
    nvg.fontSize(property.getFontSize());
    nvg.fontFace(property.getFontFace());
    nvg.textAlign(property.getFontAligntment());
}

Reply:
Or maybe add a method to the NanoVGFontProperty to set all 4 of these and one would call `fontProperty_.set(nvg)`